packages/builder/src/builder.ts

```diff
@@ -31,8 +31,12 @@ export async function createInitialContext(
     chainId,
   };
 
+  debug('define chain builder context');
+
   const settings: ChainBuilderContext['settings'] = {};
 
+  debug('get settings', preCtx);
```

Comment:
careful is this going to print out anything sensitive?

Reply:
Yeah this debug shouldnt be here actually, was mostly using it to debug an issue.